### PR TITLE
Translated to Bulgarian Language

### PR DESCRIPTION
**Pull request by ppalashturov**
Translated to Bulgarian Language

### CODE REVIEW

File: packages/Webkul/Installer/src/Resources/lang/bg/app.php [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+                'allowed-locales'             => 'Разрешени езици',
+                'application-name'            => 'Име на приложението',
+                'argentine-peso'              => 'Аржентинско песо (ARS)',
+                'australian-dollar'           => 'Австралийски долар (AUD)',
+                'bagisto'                     => 'Bagisto',
+                'bangladeshi-taka'            => 'Бангладешка така (BDT)',
+                'bahraini-dinar'              => 'Бахрейнски динар (BHD)',
+                'brazilian-real'              => 'Бразилски реал (BRL)',
+                'british-pound-sterling'      => 'Британски лири стерлинги (GBP)',
+                'bulgarian-lev'               => 'Български лев (BGN)',
```
Comment (amit-webkul) [inline packages/Webkul/Installer/src/Resources/lang/bg/app.php:643]: The `bulgarian-lev` key should be added in all locales in the installer package.

File: packages/Webkul/Installer/src/Resources/lang/bg/app.php [hunk 2]
```diff
@@ -0,0 +1,822 @@
...
+                'xml'         => 'XML',
+            ],
+
+            'arabic'                   => 'Арабски',
+            'back'                     => 'Назад',
+            'bagisto'                  => 'Bagisto',
+            'bagisto-info'             => 'е Обществен проект от',
+            'bagisto-logo'             => 'Bagisto лого',
+            'bengali'                  => 'Бенгалски',
+            'bulgarian'                => 'Български',
```
Comment (amit-webkul) [inline packages/Webkul/Installer/src/Resources/lang/bg/app.php:796]: Here also

The `bulgarian` key should be added in all locales in the installer package.

File: packages/Webkul/Installer/src/Resources/lang/en/app.php [hunk 3]
```diff
@@ -724,6 +726,10 @@
                 'yes'             => 'Yes',
             ],
 
+            'installation-wizard' => [
+                'text' => 'Installation wizard language',
+            ],
+
```
Comment (amit-webkul) [inline packages/Webkul/Installer/src/Resources/lang/en/app.php:732]: Here also 

This will be added to all local files.

File: composer.json [hunk 4]
```diff
@@ -50,7 +50,7 @@
         "bagisto/laravel-datafaker": "^2.2.0",
         "barryvdh/laravel-debugbar": "^3.8",
         "fakerphp/faker": "^1.9.1",
-        "laravel/pint": "^1.8",
+        "laravel/pint": "^1.17",
```
Comment (amit-webkul) [inline composer.json:53]: Could you please clarify why it's necessary to downgrade the Pint version in this case?

**Comment by amit-webkul** [comment]
Hello @ppalashturov ,

Once the pull request is completed, please let us know.

File: composer.json (hunk 4 above)
Comment (ppalashturov) [reply composer.json:53]: @amit-webkul

**Comment by ashishkumar-webkul** [comment]
@ppalashturov 

I kindly request you to complete the PR so we can proceed with verification and merge it.

Thank you. Looking forward to your response.

**Comment by devansh-webkul** [comment]
You can raise a new PR when its done as currently looks like PR is inactive for a long time.

**Pull request closed without merging**